ruby_project/classes/trimmer_decorator.rb

```diff
@@ -0,0 +1,6 @@
+require_relative './decorator'
+class TrimmerDecorator < Decorator
+  def correct_name
+    @nameable.correct_name[0..9]
+  end
```

Comment:
Here first this method should check whether the name is more than 10 characters or not. For that, you can use [Ternary operator ](https://syntaxdb.com/ref/ruby/ternary#:~:text=The%20ternary%20operator%20is%20used,behaving%20similar%20to%20a%20function.) and check the ``@nameable.correct_name. length > 10`` then if it's more than 10 characters return what you have now or if it's not more than 10 characters you can return it self.

Reply:
True. Thanks for the suggestion